algorithms/CPlusPlus/Strings/delete-alternate-characters.cpp

```diff
@@ -0,0 +1,35 @@
+/*
+Description: To delete alternate characters in a string
+Approach: To iterate over string and append the alternate characters in the output string
+*/
+
+// Initial Template for C++
+#include <bits/stdc++.h>
```

Comment:
No need to clutter headers when it's just header that you want. (same in other file also)

```suggestion
#include <string>
```

Reply:
Changes done!